src/test/java/com/google/api/codegen/testdata/java_main_library.baseline

```diff
@@ -1643,12 +1655,124 @@ public class LibraryServiceApi implements AutoCloseable {
    * }
    * </code></pre>
    */
-  public final ApiCallable<UpdateBookIndexRequest, Empty> updateBookIndexCallable() {
+  public final UnaryApiCallable<UpdateBookIndexRequest, Empty> updateBookIndexCallable() {
     return updateBookIndexCallable;
   }
 
   // AUTO-GENERATED DOCUMENTATION AND METHOD
   /**
+   * Test server streaming
+   * gRPC streaming methods don't have HTTP equivalent and need not have google.api.http option.
```

Comment:
-> "gRPC streaming methods don't have an HTTP equivalent and don't need to have the google.api.http option."